assets/enx-redirect/report/index.html

```diff
@@ -0,0 +1,108 @@
+{{define "report/index"}}
+
+{{$currentRealm := .realm}}
+
+<!doctype html>
+<html dir="{{$.textDirection}}" lang="{{$.textLanguage}}">
+<head>
+  {{template "report/head" .}}
+</head>
+
+<body class="tab-content">
+  <main role="main" class="container">
+    <h1>Request exposure notifications verification code from {{.realm.Name}} ({{.realm.RegionCode}})</h1>
+
+    <form method="POST" action="/report/issue" class="floating-form">
+      {{ .csrfField }}
+
+      <div id="form-area">
+
+        <div class="card mb-3 shadow-sm">
+          <div class="card-header">Enter the date of your COVID-19 test</div>
+          <div class="card-body">
+            <div class="form-row">
+              <div class="form-group col-md-6">
+                <label for="testDate">{{t $.locale "codes.issue.testing-date-label"}}</label>
+                <input type="date" id="test-date" name="testDate" min="{{.minDate}}" max="{{.maxDate}}" class="form-control" {{if $currentRealm.RequireDate}}required{{end}} />
+              </div>
+            </div>
+          </div>
+        </div>
+
+        <div class="card mb-3 shadow-sm">
+          <div class="card-header">Enter the date of your COVID-19 symptoms</div>
+          <div class="card-body">
+            <div class="form-row">
+              <div class="form-group col-md-6">
+                <label for="symptomDate">Leave blank if you have not experienced symptoms</label>
+                <div class="input-group">
+                  <input type="date" id="symptom-date" name="symptomDate" min="{{.minDate}}" max="{{.maxDate}}" class="form-control" {{if $currentRealm.RequireDate}}required{{end}} />
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+
+        <div class="card mb-3 shadow-sm">
+          <div class="card-header">Phone Number</div>
+          <div class="card-body">            
+            <div class="form-row">
+              <label for="phone">Enter your mobile phone number. Your Exposure Notifications
+                verification code from {{.realm.Name}} will be send to you via text message.
+              </label>
+            </div>
+            <div class="form-row">
+              <div class="input-group">
+                <input type="tel" id="phone" name="phone" class="form-control" autocomplete="off" class="w-100" />
```

Comment:
```suggestion
                <input type="tel" id="phone" name="phone" class="form-control" autocomplete="off" class="d-block" />
```

If you're trying to make it fill the box